modules/load-balancer-target-group/main.tf

```diff
@@ -4,7 +4,16 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = ">= 0.12"
+  # Source is required for required_providers in TF 13, but is only compatible with TF 12 starting 0.12.26.
+  required_version = ">= 0.12.26"
+
+  # Using provider v3.0 syntax for aws_alb_listener_rule condition blocks.
```

Comment:
I thought this syntax existed in 2.x too, and it was merely the old syntax was deprecated in 2.x, and now an error in 3.x? If so, then we don't need to strictly require 3.x here.

Reply:
Ah you are right! Just ran a quick test forcing provider v2 (2e78176fb5993824d41ab345b953a2e3a8fd02d1) and it worked. I relaxed the constraint: d00c5c799d8721eeeef27850d931d73aa3894f09.